src/NvGet.Tests/Resources/project_properties.json

```diff
@@ -0,0 +1,4 @@
+﻿[
+  { "PropertyName": "UnoExtensionsVersion", "PackageId": "Uno.Extensions.Core" },
+  { "PropertyName": "UnoVersion", "PackageId": "Uno.WinUI" }
+]
```

Comment:
Is there an in-progress test work?

Reply:
No there isn't and to be fair we should add a test solution that can be used for verifying updater features